crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -475,11 +478,16 @@ impl PipelineCache {
     pub fn get_bind_group_layout(
         &self,
         bind_group_layout_descriptor: &BindGroupLayoutDescriptor,
-    ) -> BindGroupLayout {
-        self.bindgroup_layout_cache
-            .lock()
-            .unwrap()
-            .get(&self.device, bind_group_layout_descriptor.clone())
+    ) -> &BindGroupLayout {
```

Comment:
As the lifetime of the returned `&BindGroupLayout` is being created out of thin air here, it is important to be explicit with the lifetime relationships here.